src/wp-includes/class-wp-scripts.php

```diff
@@ -390,8 +390,12 @@ public function do_item( $handle, $group = false ) {
 			return true;
 		}
 
+		$strategy = $this->get_eligible_loading_strategy( $handle );
+		if( '' !== $strategy ) {
+			$strategy = ' ' . $strategy;
+		}
 		$tag  = $translations . $cond_before . $before_handle;
-		$tag .= sprintf( "<script%s src='%s' id='%s-js'></script>\n", $this->type_attr, $src, esc_attr( $handle ) );
+		$tag .= sprintf( "<script%s src='%s' id='%s-js'%s></script>\n", $this->type_attr, $src, esc_attr( $handle ), $strategy );
```

Comment:
Let's go ahead and update this to use numbered placeholders and split each replacement to new lines, for readability. The `src` attribute should probably be escaped here as well, so adding it in.

```suggestion
		$tag .= sprintf( 
			"<script%1$s src='%2$s' id='%3$s-js'%4$s></script>\n",
			$this->type_attr,
			esc_url($src),
			esc_attr( $handle ),
			$strategy
		);
```